src/components/Pages/Acknowledgements.tsx

```diff
@@ -1,5 +1,6 @@
 import styles from './Pages.module.scss';
 
+// REVIEW: Checked around and this component is not being used. Maybe remove?
```

Comment:
Thanks for flagging this! If you see anything like that, feel free to remove the file rather than commenting on it